ProcessMaker/Providers/ProcessMakerServiceProvider.php

```diff
@@ -407,8 +407,6 @@ public static function getQueryTime(): float
      */
     public static function setPackageBootStart(string $package, $time): void
     {
-        $package = ucfirst(\Str::camel(str_replace(['ProcessMaker\Packages\\', '\\'], '', $package)));
-
         self::$packageBootTiming[$package] = [
             'start' => $time,
```

Comment:
validate if $time is valid
example
```
if ($time < 0) {
        throw new \InvalidArgumentException('Time must be a positive value');
    }
```

Reply:
A log entry was added since it is only a developer tool